web/__init__.py

```diff
@@ -98,6 +135,86 @@ def app_api_version():
     }
 
 
+def allowed_file(filename, ALLOWED_EXTENSIONS):
```

Comment:
a parameter is not a constant. No `SCREAMING_SNAKE_CASE` here.

Reply:
true, I copy/pasta'd here :)